src/docfx/Errors.cs

```diff
@@ -126,6 +126,13 @@ public static Error NullArrayValue(SourceInfo? source, string name)
         /// Behavior: ❌ Message: ❌
         public static Error ViolateSchema(SourceInfo? source, string message)
             => new(ErrorLevel.Error, "violate-schema", $"{message}", source);
+
+        /// <summary>
+        /// Path string contains invalid path chars.
+        /// </summary>
+        /// Behavior: ❌ Message: ❌
+        public static Error PathInvalid(SourceInfo? source, string message)
+            => new(ErrorLevel.Warning, "path-invalid", $"{message}", source);
```

Comment:
```suggestion
            => new(ErrorLevel.Warning, "path-invalid", message, source);
```

Reply:
@herohua We use interpolated string here because the parameter of 'Error' is 'FormattableString' type:
![image](https://user-images.githubusercontent.com/25164547/154641316-47e17a30-08df-41c3-a4fa-4c2bbf17fba7.png).

I have checked the diffs in failed regression tests results, they are all unrelated diffs. I don't have permission to merge PR if any PR check in failed state, could you please help merge the PR if it's ready to merge?

Reply:
You'll need to move this message template here: '$"Path {path} contains invalid chars {string.Join(", ", invalidPathChars.Select(c => $"'{c}'"))}.'